tools/compile-proto-file/Main.hs

```diff
@@ -38,6 +38,9 @@ parseArgs = info (helper <*> parser) (fullDesc <> progDesc "Compiles a .proto fi
         <> metavar "DIR"
         <> help "Output directory path where generated Haskell modules will be written (directory is created if it does not exist; note that files in the output directory may be overwritten!)"
 
+    unwrap = switch $
+      long "unwrap"
+        <> help "If this flag is set, Protobuffer types are properly translated into Haskell Maybe types."
```

Comment:
Minor suggestion: this help text may need to provide a little more context. A user unfamiliar with the history of the tooling I might think "why _wouldn't_ I want to have the proper translation?"

In fact, since this change will entail a package version bump anyway, I think it might make sense to have the new behavior be the default, with the flag provided to opt into the old behavior. If you decide to go that way, perhaps the help text phrasing for the `--use-legacy-wrapper-types` option (e.g.) could be something like "Use legacy newtype wrapping for protobuf wrapper types (e.g. Int32Value) in generated Haskell code (provided for backwards compatibility)" or something along those lines.

Reply:
Okay got it. I'm just reading this comment, but I will implement this in my next commit. I like this idea.